superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -298,11 +300,18 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
             <div className="helper">
               {t('Refer to the ')}
               <a
-                href={conf?.SQLALCHEMY_DOCS_URL ?? ''}
+                href={
+                  SQLALCHEMY_DOCS_URL ??
+                  'https://docs.sqlalchemy.org/en/13/core/engines.html'
+                }
                 target="_blank"
                 rel="noopener noreferrer"
               >
-                {conf?.SQLALCHEMY_DISPLAY_TEXT ?? ''}
+                {st(
+                  'DATABASE_MODAL.SQLALCHEMY_DISPLAY_TEXT',
+                  SQLALCHEMY_DISPLAY_TEXT,
+                  'SQLAlchemy docs',
+                )}
```

Comment:
if you wanted pybabel to pick this up for extraction, you could try add `st` as a keyword.